lib/notifiex/services/discord.ex

```diff
@@ -14,16 +16,19 @@ defmodule Notifiex.Service.Discord do
   `options` should include the following:
   * `webhook`: Webhook URI. (required)
   """
-  @spec call(map, map) :: {:ok, binary} | {:error, {atom, any}}
-  def call(payload, options) when is_map(payload) and is_map(options) do
+  @spec call(payload(), payload()) :: Notifiex.result()
+  def call(payload = _, options = _)
+      when not (is_map(payload) and is_map(options) and is_map_key(payload, :content) and
+                  is_map_key(options, :webhook)),
```

Comment:
Is there a reason why this is split into 2 lines? Is it due to max line length?

Reply:
Yes, for me it's automatically done by the formatter